frontend/app/src/screens/object-items/object-items-cell.tsx

```diff
@@ -0,0 +1,76 @@
+import { getObjectDetailsUrl2 } from "@/utils/objects";
+import { Link, LinkProps } from "react-router-dom";
+import {
+  getDisplayValue,
+  RelationshipManyType,
+  RelationshipOneType,
+} from "@/utils/getObjectItemDisplayValue";
+import { AttributeSchema, RelationshipSchema } from "@/screens/schema/types";
+import { Badge } from "@/components/ui/badge";
+import { classNames } from "@/utils/common";
+import { HTMLAttributes } from "react";
+
+type ObjectItemsCellProps = {
+  row: any;
+  attribute:
+    | (RelationshipSchema & { isRelationship: true; paginated: boolean })
+    | (AttributeSchema & { isAttribute: boolean });
+};
+
+export const ObjectItemsCell = ({ row, attribute }: ObjectItemsCellProps) => {
+  if ("isRelationship" in attribute && attribute.isRelationship) {
+    if (attribute.cardinality === "one") {
+      return <RelationshipOneCell data={row[attribute.name]} />;
+    }
+
+    if (attribute.cardinality === "many") {
+      return <RelationshipManyCell data={row[attribute.name]} />;
+    }
+  }
+
+  const url = getObjectDetailsUrl2(row.__typename, row.id);
+
+  return <LinkCell to={url}>{getDisplayValue(row, attribute)}</LinkCell>;
+};
+
+export const TextCell = ({ className, ...props }: HTMLAttributes<HTMLSpanElement>) => {
+  return <span className={classNames("p-2 text-xs whitespace-nowrap", className)} {...props} />;
+};
+
+export const LinkCell = ({ className, children, ...props }: LinkProps) => {
+  return (
+    <Link className={classNames("h-full flex items-center", className)} {...props}>
```

Comment:
why not using the link ui component like discussed previously, so all links remain the same?

Reply:
We use a "hack" to make each cell in a row a link to the related node. However, I don't want the entire row to be underlined when we hover over it.

So having the component link underline on hover is not ideal in this case. And that's why I did not use it